hdf/src/hconv.h

```diff
@@ -58,7 +58,7 @@
 /* CONSTANT DEFINITIONS                                                      */
 /*****************************************************************************/
 /* Generally Big-Endian machines */
-#if !defined(INTEL86) && !defined(__x86_64__)
+#ifdef __BIG_ENDIAN__
```

Comment:
Is that the same define as CMake? I thought there was a configure variable; H4_WORDS_BIGENDIAN used by both autotools and CMake.